ui/src/components/credentials/CredentialDetails.tsx

```diff
@@ -216,6 +224,11 @@ export function CredentialDetails() {
                       text={expiresAt ? formatDate(expiresAt) : "-"}
                     />
 
+                    <Detail
+                      label="Refresh Service"
+                      text={refreshService ? " " + refreshService.id : "-"}
```

Comment:
Instead of doing this, we should adapt the `Detail` component to cover this new requirement.

Reply:
Fixed. I've updated the <Detail> component adding an href parameter. If this parameter is present, the <Detail> will convert that text into a link like <a href="{href}">{text}</a>